tests/Kros.AspNetCore.Tests/Authorization/GatewayAuthorizationMiddlewareShould.cs

```diff
@@ -148,14 +177,16 @@ private static (IHttpClientFactory, GatewayAuthorizationMiddleware) CreateMiddle
 
             httpClientFactory.CreateClient("JwtAuthorizationClientName").Returns(fakeHttpClient);
 
-            var middleware = new GatewayAuthorizationMiddleware((c) =>
-            {
-                return Task.CompletedTask;
-            }, new GatewayJwtAuthorizationOptions()
+            var gatewayJwtAuthorizationOptions = new GatewayJwtAuthorizationOptions()
             {
                 AuthorizationUrl = "http://authorizationservice.com",
                 CacheSlidingExpirationOffset = offset
-            });
+            };
+            gatewayJwtAuthorizationOptions.IgnoredPathForCache.AddRange(ignoredPathForCache);
+            var middleware = new GatewayAuthorizationMiddleware((c) =>
```

Comment:
You can simplify `var middleware = new GatewayAuthorizationMiddleware((c) => Task.CompletedTask, gatewayJwtAuthorizationOptions);`